src/js/components/user/login.tsx

```diff
@@ -50,23 +53,79 @@ const Login = ({ user }: { user: User | null }) =>
   user ? (
     <Redirect to={routes.home()} />
   ) : (
-    <div className="slds-align_absolute-center">
-      <div className="slds-text-longform slds-p-around_x-large">
-        <h1 className="slds-text-heading_large">
-          {i18n.t('Welcome to MetaShare!')}
-        </h1>
-        <p>
-          This is a stub. It will eventually be replaced with an{' '}
-          <a href="https://www.lightningdesignsystem.com/components/welcome-mat/">
-            SLDS Welcome Mat Component
-          </a>{' '}
-          once the{' '}
-          <a href="https://github.com/salesforce/design-system-react/issues/1876">
-            React implementation
-          </a>{' '}
-          is complete.
-        </p>
-        <LoginButton />
+    <div
+      className="slds-welcome-mat
+        slds-welcome-mat_info-only
+        welcome-container"
+    >
+      <div className="slds-welcome-mat__content slds-grid welcome-inner">
+        <div
+          className="slds-welcome-mat__info
+            slds-size_1-of-1
+            slds-medium-size_1-of-2"
+          style={{ backgroundImage: `url(${welcomeMatBG})` }}
+        >
+          <div className="slds-welcome-mat__info-content">
+            <h2 className="slds-welcome-mat__info-title">
+              {i18n.t('Welcome to MetaShare!')}
+            </h2>
+            <div
+              className="slds-welcome-mat__info-description
+                slds-text-longform"
+            >
+              <p>
+                {i18n.t(
+                  'Welcome to MetaShare, the web-based tool for collaborating on Salesforce projects.',
+                )}
+              </p>
+            </div>
+            <div className="slds-welcome-mat__info-actions">
+              <LoginButton />
+            </div>
+          </div>
+        </div>
+        <div
+          className="slds-welcome-mat__tiles
+            slds-size_1-of-1
+            slds-medium-size_1-of-2
+            slds-welcome-mat__tiles_info-only
+            slds-grid
+            slds-grid_vertical
+            slds-grid_align-center
+            slds-p-left_xx-large
+            slds-p-right_xx-large"
+        >
+          <WelcomeMatTile
```

Comment:
Clearly these tiles need content - but this first "welcome" tile feels likely redundant with the "welcome" info we already have on the left?

Reply:
Agreed. Everything here is placeholder as of now including the topics for the blocks of text. I do think if there isn't an obvious set of instructions, we can answer the who, what, when/where, why and mostly how types of questions someone might have. If this isn't helpful, then we would need to re-think this side of the mat. I've never written copy for a client though so if that is on us, I think I'd assign that to someone on our team that does have a writing background. Or maybe the SF team wants to fill these blocks in? I'm not sure where we stand on that.

Reply:
Those are good questions. I'd say given this is an in-development app, let's merge this and then discuss copy (and who should write it) with Jason and David?